README.md

```diff
@@ -51,6 +67,20 @@ npm run dev -- --host
 
 The QR code shown will allow you to quickly access the app.
 
+### DB
+
+To create migrations to the database, first update the Prisma schema at ./prisma/schema.prisma as required, then run the below to generate the corresponding SQL migration and to apply it to the database:
+
+```bash
+npx prisma migrate dev
+```
+
+The same command is also used to apply migrations that already exist in ./prisma/migrations but which have not been applied to the database.
```

Comment:
- Updating README to add "You should commit both the Prisma schema and the migration file to Git."
- Will look into the precise process for how migrations are done in deployment, but probably it's a GitHub Action that runs the command `prisma migrate deploy` (as opposed to the dev-specific command above).

Reply:
Yeah, we just need to run `prisma migrate deploy` in a github action, and remember to set the DATABASE_URL env var (which we can do in github per-environment).

https://www.prisma.io/docs/orm/prisma-client/deployment/deploy-database-changes-with-prisma-migrate

Reply:
I think it makes most sense to try that out when we have some environments up, with databases running, which can see whether or not they correctly get migrated.

Reply:
Note to self for when I'm working on this:

It might be the case that we need to explicitly re-generate the prisma client (using [prisma generate](https://www.prisma.io/docs/orm/prisma-client/setup-and-configuration/introduction#5-evolving-your-application)) after running `prisma migrate deploy`, since that command "[does not reset the database or generate artifacts (such as Prisma Client)](https://www.prisma.io/docs/orm/prisma-migrate/workflows/development-and-production#customizing-migrations)"